requirements.txt

```diff
@@ -2,11 +2,13 @@ alembic==0.9.9
 Flask==1.0
 Flask-Migrate==2.1.1
 Flask-Script==2.0.6
-Flask-SQLAlchemy==2.3.2
+Flask-SQLAlchemy==2.5.0
 Flask-WTF==0.14.3
-psycopg2
-pylint==1.8.3
+pylint==2.10.2
 SQLAlchemy>=1.3.0
-Flask-pyoidc
+Flask-pyoidc==3.2.0
 pytz==2018.3
 Flask-login
+psycopg2-binary
+slack_sdk
+Flask-Mail
```

Comment:
These should _REALLY_ be versioned...

Reply:
https://github.com/ComputerScienceHouse/csh-rideboard/pull/22/commits/40353a711e7622223f6486135ed0c1cb154044bd